cmd/juno/juno.go

```diff
@@ -327,6 +330,7 @@ func NewCmd(config *node.Config, run func(*cobra.Command, []string) error) *cobr
 	junoCmd.Flags().String(cnCoreContractAddressF, defaultCNCoreContractAddressStr, networkCustomCoreContractAddressUsage)
 	junoCmd.Flags().IntSlice(cnUnverifiableRangeF, defaultCNUnverifiableRange, networkCustomUnverifiableRange)
 	junoCmd.Flags().String(ethNodeF, defaultEthNode, ethNodeUsage)
+	junoCmd.Flags().Bool(noEthNodeF, defaultNoEthNode, noEthNodeUsage)
```

Comment:
You should make `no-eth-node` and `eth-node` mutually exclusive so that both cannot be passed at the same time. Otherwise, some can pass both these flags which may lead to unexpected behaviour in node.go.

Reply:
Agree, added